src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelGlobalConstants.java

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import java.io.Serializable;
+
+/**
+ * Global constants for coverage model package classes
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class CoverageModelGlobalConstants implements Serializable {
```

Comment:
I don't think there's anything to be gained from 17 additional arguments... the user can specify a single output path and rename/move things around if needed (which is not recommended to keep the provenance intact).